code/game/objects/items/weapons/material/foam.dm

```diff
@@ -37,7 +37,7 @@
 	hitsound = 'sound/effects/bodyfall3.ogg'
 
 /obj/item/material/twohanded/spear/foam/Initialize(mapload, material_key)
-	return ..(mapload,"foam")
```

Comment:
no
this needs to **return**

Reply:
That's odd. I didn't remove this line in my code. I'm looking over at it right now and it's still there.
![image](https://user-images.githubusercontent.com/5233182/215633183-3b5d0efd-0235-461b-a355-e5333db7390a.png)

Reply:
Disregard. I'm not sure why the return was removed. I don't remember doing that. Easy fix.